pom.xml

```diff
@@ -220,8 +220,6 @@
           <overview>${basedir}/src/main/javadoc/overview.html</overview>
           <javadocDirectory>${basedir}/src/main/javadoc</javadocDirectory>
           <reportOutputDirectory>${project.build.directory}/apidocs</reportOutputDirectory>
-          <!-- This will avoid adding a folder level for the base module name. Note: Support removed in JDK 13 -->
-          <additionalJOption>--no-module-directories</additionalJOption>
```

Comment:
JDK 13+ removed this option, but it is still needed in JDK11 (which is used for compiation _and javadoc_). Wouldn't removing this force the addition of a module level at the javadoc? It will be a must in future versions of the JDK but, for now…

Reply:
Sorry this was caused by me using the wrong JDK to build. I have reverted this.